napari/layers/vectors/_slice.py

```diff
@@ -92,23 +93,7 @@ def __call__(self) -> _VectorSliceResponse:
             request_id=self.id,
         )
 
-    def _get_out_of_display_slice_data(self, not_disp, not_disp_indices):
```

Comment:
This is un-used, so I figured easier to just delete it instead of trying to type it.